Samples/Tutorials/EventHorizon/Environment/docker-compose.yml

```diff
@@ -24,6 +26,8 @@ services:
       - ./producer-platform.json:/app/.dolittle/platform.json
       - ./producer-resources.json:/app/.dolittle/resources.json
       - ./producer-event-horizon-consents.json:/app/.dolittle/event-horizon-consents.json
+    environment:
+      - Logging__Console__FormatterName=""
```

Comment:
This shouldn't be necessary when we set it in the docker image?

Reply:
Yes, shouldn't be necessary